open-metadata-implementation/access-services/governance-engine/governance-engine-server/pom.xml

```diff
@@ -75,11 +75,10 @@
             <artifactId>governance-engine-api</artifactId>
         </dependency>
 
-        <!-- Spring for calling REST APIs -->
-
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-core</artifactId>
+            <version>5.2.4.RELEASE</version>
```

Comment:
This version identification needs removing -- versions should only be in dependencyManagement in our top level pom

Reply:
thanks! I removed it in the last commit.